cpmpy/solvers/pysat.py

```diff
@@ -93,12 +90,25 @@ def supported():
             # while we need the 'python-sat' package, some more checks:
             from pysat.formula import IDPool
             from pysat.solvers import Solver
+
+            # try to import pypblib once
+            if CPM_pysat._pb is None:
+                from pysat import card
+                CPM_pysat._card = card  # native
+                try:
+                    from pysat import pb  # require pypblib
+                    CPM_pysat._pb = pb
+                except (ModuleNotFoundError, NameError):  # pysat returns the wrong error type (latter i/o former)
+                    CPM_pysat._pb = False  # not installed, avoid reimporting
```

Comment:
you set it to False but the check above online 95 checks for None... so should be None?

Also your check in the pb function below should be consistent with the choice you will make

Reply:
Explained in other comment